src/test/kotlin/no/nav/syfo/api/endpoints/PodApiSpek.kt

```diff
@@ -16,86 +19,95 @@ object PodApiSpek : Spek({
     val database = TestDatabase()
     val databaseNotResponding = TestDatabaseNotResponding()
 
-    describe("Successful liveness and readiness checks") {
-        with(TestApplicationEngine()) {
-            start()
-            application.routing {
+    fun ApplicationTestBuilder.setupPodApi(database: DatabaseInterface, applicationState: ApplicationState) {
+        application {
+            routing {
                 podEndpoints(
-                    applicationState = ApplicationState(
-                        alive = true,
-                        ready = true
-                    ),
+                    applicationState = applicationState,
                     database = database,
                 )
             }
+        }
+    }
+
+    describe("Successful liveness and readiness checks") {
+        it("Returns ok on is_alive") {
+            testApplication {
```

Comment:
For tester som skal gjøre requests mot appens endepunkter må man ha oppsett ala dette (ref: https://ktor.io/docs/migrating-3.html#testing) i stedet for `with(TestApplicationEngine())` osv som vi hadde tidligere

Reply:
Så kan man gjøre requests med `client.get(..)` i stedet for `with(handleRequest(...))`